src/world/Objects/Units/Unit.cpp

```diff
@@ -75,7 +75,7 @@ Unit::Unit() :
 #endif
 
     // Zyres: initialise here because multiversion differences
-    std::fill_n(PctPowerRegenModifier, TOTAL_PLAYER_POWER_TYPES, 1);
+    std::fill_n(PctPowerRegenModifier, TOTAL_PLAYER_POWER_TYPES, '1.0f');
```

Comment:
Without quotes. Like this (PctPowerRegenModifier, TOTAL_PLAYER_POWER_TYPES, 1.0f);

Reply:
cleared cache and compiled (without '')
https://github.com/fmtlib/fmt/blob/master/include/fmt/chrono.h#L1692 I looked at the option from 'format' there just with quotes